elink, esearch: add page (#8957)
tldr-pages/tldr: pages/linux/esearch.md

# esearch

> > Esearch performs a new Entrez search using terms in indexed fields.

Reviewer: ```suggestion
> Esearch performs a new Entrez search using terms in indexed fields (It is part of the edirect package).
```
Is this fine?

Author: I separated it out and so technically there are three sentences in the description.  One of the sentences says it is part of the edirect package.

> > Esearch performs a new Entrez search using terms in indexed fields.

Reviewer 2: ```suggestion
> Perform a new Entrez search using terms in indexed fields.
```

> It requires a `-db` argument for the database name and uses `-query` for the search terms.

> > It is part of the edirect package: <https://www.ncbi.nlm.nih.gov/books/NBK179288/>.

Reviewer: ```suggestion
> More information: <https://www.ncbi.nlm.nih.gov/books/NBK179288/>.
```

- Search pubmed:

> `esearch -db pubmed -query "selective serotonin reuptake inhibitor"`

Reviewer 2: ```suggestion
`esearch -db pubmed -query "{{selective serotonin reuptake inhibitor}}"`
```

> - Search nucleotide:

Reviewer 2: could we find a better explanation of the example?
```suggestion
- Print nucleotide in X format matching ... query:
```

> `esearch -db nuccore -query "insulin [PROT] AND rodents [ORGN]"`

Reviewer 2: ```suggestion
`esearch -db nuccore -query "{{insulin [PROT] AND rodents [ORGN]}}"`
```

# esearch

> Perform a new Entrez search using terms in indexed fields.

> > It requires a `-db` argument for the database name and uses `-query` for the search terms.

Reviewer 2: we can only accept up to 2 lines for command descriptions. Also I think this disclaimer is easy to understand. IMO it doesn't need an explanation 
```suggestion
```

Author: That's fine

> > It is part of the edirect package.

Reviewer 2: ```suggestion
> It is part of the `edirect` package.
```
